inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_jar.adoc

```diff
@@ -73,11 +72,19 @@ $ chown www-data:www-data /srv/inception/inception.jar
 $ chmod +x /srv/inception/inception.jar
 ----
 +
-* Create a symlink from `/etc/init.d` to the `/srv/inception/inception.jar`:
+* Create a file in `/etc/systemd/system/inception.service` with the following content:
 +
 [source,bash]
 ----
-$ ln -s /srv/inception/inception.jar /etc/init.d/inception
+[Unit]
+Description=INCEpTION
+
+[Service]
+ExecStart= /srv/inception/inception.jar
```

Comment:
Is the space between `=` and `/` intentional?

Reply:
Fixed